src/Sylius/Component/Core/Taxation/Applicator/OrderItemUnitsTaxesApplicator.php

```diff
@@ -24,20 +24,8 @@
 
 class OrderItemUnitsTaxesApplicator implements OrderTaxesApplicatorInterface
 {
-    private CalculatorInterface $calculator;
-
-    private AdjustmentFactoryInterface $adjustmentFactory;
-
-    private TaxRateResolverInterface $taxRateResolver;
-
-    public function __construct(
-        CalculatorInterface $calculator,
-        AdjustmentFactoryInterface $adjustmentFactory,
-        TaxRateResolverInterface $taxRateResolver
-    ) {
-        $this->calculator = $calculator;
-        $this->adjustmentFactory = $adjustmentFactory;
-        $this->taxRateResolver = $taxRateResolver;
+    public function __construct(private CalculatorInterface $calculator, private AdjustmentFactoryInterface $adjustmentFactory, private TaxRateResolverInterface $taxRateResolver)
+    {
```

Comment:
```suggestion
    ```suggestion
    public function __construct(
        private CalculatorInterface $calculator,
        private AdjustmentFactoryInterface $adjustmentFactory,
        private TaxRateResolverInterface $taxRateResolver
    ) {
```
```